rise_set/astrometry.py

```diff
@@ -9,10 +7,39 @@
 The code uses SLALIB for the heavy lifting. The rise/set/transit algorithms here
 are implementations of Astronomical Algorithms, Ch. 14 (Jean Meeus).
 
-
-Author: Eric Saunders (esaunders@lcogt.net)
-May 2010
-'''
+The input to most astrometry calculation involves a target dictionary containing the
+necessary target data. This package contains functions to help creating a target
```

Comment:
"This package contains functions for creating a target dictionary and validating required fields for the supported target types."